### DISCUSSION TITLE
minor spelling typo

### DISCUSSION THREAD

**Discussion by DickBaker** [cake-build/cake#4333] [Extension Q&A] [answered]
on page https://cake-contrib.github.io/CakeContrib.Guidelines/ please replace "guildelines" by "guidelines"

**Comment by pascalberger** [comment]
Thanks for the report. Are you in a position to provide a PR? The source file is here: https://github.com/cake-contrib/CakeContrib.Guidelines/blob/develop/docs/input/index.cshtml

**Reply by DickBaker** [reply, marked as answer]
@pascalberger Hi, I just submitted PR #352 (https://github.com/cake-contrib/CakeContrib.Guidelines/pull/352)

whilst I was exploring I noticed that footer for Cake states
Website generated by [Wyam](http://wyam.io/)

but when I browsed that URL I got
![WyamGone](https://github.com/user-attachments/assets/7d1b41c2-438d-4a48-8a7f-d21857406278)

so needs checking your dependencies in case someone forgot to domain renew !

**Reply by pascalberger** [reply]
Thanks, the footer link will be fixed with https://github.com/cake-build/website/pull/3035

**Reply by DickBaker** [reply]
great, otherwise you should contact swissgrc for Third Party Risk Management! (`<grin/>`)
